configtx/application.go

```diff
@@ -39,6 +37,18 @@ type ApplicationOrg struct {
 	name     string
 }
 
+// ApplicationOrgMSP encapsulates the configuration functions used to modify an application organization's MSP.
```

Comment:
This type is unused now and can be deleted.

Reply:
🛠️